app/registries/foreman/access_permissions.rb

```diff
@@ -314,6 +314,7 @@
                                       :"api/v2/common_parameters" => [:destroy],
                                       :"api/v2/parameters" => [:destroy, :reset],
                                   }
+    map.permission :create_lookup_values, { :lookup_values => [:index] }
```

Comment:
I'm not sure if it's needed just yet. Where it can be used out of https://github.com/theforeman/foreman_ansible/pull/633 scope? I mean, `:create_lookup_values` permission is defined in `permission_list`, and than being used in that PR for `Ansible Role Manager` role.

AFAIU, this particular line means that if one wants to use lookup_values controller (not the API one) and call `index` action on that, the `:create_lookup_values` permission will be needed for that. Which is also not quite right, since I'd expect `:view_lookup_values` permission instead.

Reply:
You are correct, it's not needed. I didn't test that properly.